sse2neon.h

```diff
@@ -8546,7 +8535,11 @@ FORCE_INLINE uint32_t _mm_crc32_u8(uint32_t crc, uint8_t v)
     crc = __crc32cb(crc, v);
 #else
     crc ^= v;
-    crc = (crc >> 8) ^ _sse2neon_crc32_tbl[crc & 0xFF];
+
+    // Apply half-byte comparision algorithm for the best ratio between
```

Comment:
Since you already shrink the size of the table, move the table inside the function body.

Reply:
Done.